### PR TITLE
Document proposed changes in minetest game.

### PR DESCRIPTION
**Pull request by red-001**
See https://github.com/minetest/minetest_game/pull/809. This documents the changes in that PR in lua_api.txt

### CODE REVIEW

File: doc/lua_api.txt [hunk 1]
```diff
@@ -3513,7 +3513,7 @@ Definition tables
         ^ No return value ]]
 
         on_blast = func(pos, intensity), --[[
-        ^ intensity: 1.0 = mid range of regular TNT
+        ^ intensity: 1.0 = Right next to regular TNT. Is calculated as an inverse-square of distance form TNT.
```
Comment (rubenwardy) [inline doc/lua_api.txt]: `form` -> `from`

**Comment by paramat** [comment]
Perhaps a better wording:
Intensity is 1 at a distance of 1 from blast centre, and falls as the inverse-square of distance.

**Comment by est31** [comment]
Please also document which type of metric is used for the "distance" (similar to #3192).

**Comment by ShadowNinja** [comment]
Isn't it `1` at a distance of `0`?

**Comment by paramat** [comment]
With inverse-square it's infinite at distance 0 so it might be limited to it's maximum value at distance 1.
Intensity is likely to be changed though and scaled with tnt power.
I assume the distance type is 'radial'.

**Comment by ShadowNinja** [comment]
It should definitely not be infinite at the center.  The center may be very intense, but it will never be infinite.  I'd just do `1/(dist*dist+1)` or so.

**Comment by paramat** [comment]
WIP because the behaviour of intensity is still being decided in another PR.

**Comment by paramat** [comment]
Closing as changes to TNT are being reworked.

**Pull request closed without merging**